src/components/Avatar/Avatar.tsx

```diff
@@ -26,7 +26,10 @@ export interface AvatarProps extends VibeComponentProps {
   withoutTooltip?: boolean;
   icon?: SubIcon;
   type?: AvatarType;
+  /** ClassName for a text type of AvatarContent */
   textClassName?: string;
+  /** ClassName for a div-wrapper of AvatarContent */
```

Comment:
Maybe can be confusing because avatar can be also square?
Maybe contentClassName?

Reply:
Orr proposed `avatarContentWrapperClassName`. Wdyt?